source/Octopus.Tentacle.Tests/Capabilities/CapabilitiesServiceV2Fixture.cs

```diff
@@ -0,0 +1,20 @@
+using FluentAssertions;
+using NUnit.Framework;
+using Octopus.Tentacle.Services.Capabilities;
+
+namespace Octopus.Tentacle.Tests.Capabilities
+{
+    public class CapabilitiesServiceV2Fixture
+    {
+        [Test]
+        public void CapabilitiesAreReturned()
+        {
+            var capabilities = new CapabilitiesServiceV2()
+                .GetCapabilities()
+                .SupportedCapabilities;
+
+            capabilities.Should().Contain("ScriptService");
```

Comment:
Were we not going to return the interfaces e.g. `nameof(IScriptService)`.

This allows a caller to also use the contract names for discovery and reduces a few magic strings.